shared/getting-started/getDeviceOnDash/raspberry-pi.md

```diff
@@ -11,7 +11,7 @@ You should now be ready to deploy some code!
 {{>"getting-started/sd-card-recommendations"}}
 
 ##### Help! My device won't show up.
-If your device still hasn't shown up on your dashboard after 10 minutes, something is definitely wrong. First check that you entered the WiFi credentials correctly and ensure that your network meets these [basic requirements][networkRequirements]. It may also be worth checking the [LED error notifications][errorNotifications].
+If your device still hasn't shown up on your dashboard after 10 minutes, something is definitely wrong. First check that you entered the WiFi credentials correctly ([see this][wifiHelp] if you need help fixing your credentials) and ensure that your network meets these [basic requirements][networkRequirements]. It may also be worth checking the [LED error notifications][errorNotifications].
```

Comment:
I’m nit picking, but it is generally best for links to be sort of self-describing, avoiding words like “click here”, “check this”, etc. I would write this instead as:

```suggestion
If your device still hasn't shown up on your dashboard after 10 minutes, something is definitely wrong. First check that you entered the WiFi credentials correctly (if you need help fixing your credentials, see [WiFi Help][wifiHelp]) and ensure that your network meets these [basic requirements][networkRequirements]. It may also be worth checking the [LED error notifications][errorNotifications].
```

Reply:
Thank you @dfunckt . You make a great point. I did a little feeler to just Raspberry Pis just to see how things work and learn. I make an adjustment when I address all the devices at large.